taxonomies/categories.txt

```diff
@@ -46033,6 +46040,14 @@ en:Fruit skyrs, Fruit skyr
 fr:Skyrs aux fruits, Skyr aux fruits
 nl:Skyrs met vruchten
 
+<en:Fermented dairy desserts
+fi:Viili
+sv:Filmjölk
```

Comment:
```suggestion
fr:Viili
ko:빌리
nl:Viili
pl:Viili
sv:Filbunke, Filmjölk
wikidata:en:Q14732517
description:en:Fermented Finnish Milk Product
```

For new ingredients, you can search on wikidata and if it exists, you can add all translations
Also, for new ingredients, you can try to find the EN translation and put it on top